types/jest/jest-tests.ts

```diff
@@ -1498,6 +1498,32 @@ test.each([[1, 1, 2], [1, 2, 3], [2, 1, 3]])(
     5000
 );
 
+const constCases: [['a', 'b', 'ab'], ['d', 2, 'd2']] = [['a', 'b', 'ab'], ['d', 2, 'd2']];
+test.each(constCases)('%s + %s', (s1, s2, expected) => {
```

Comment:
This&nbsp;should test&nbsp;that the&nbsp;`args`&nbsp;array is&nbsp;correctly a&nbsp;tuple.

```suggestion
test.each(constCases)('%s + %s', (...args) => {
    args; // $ExpectType ["a", "b", "ab"] | ["d", 2, "d2"]
    const [s1, s2, expected] = args;
```

Reply:
Something really happen when I try to do that.
![image](https://user-images.githubusercontent.com/1251825/67774276-fb30ff80-fa54-11e9-9a05-9f32ca66333e.png)
No matter what i put, it fails.